paper-shadow.html

```diff
@@ -23,34 +23,90 @@
 -->
 
 <link href="../polymer/polymer.html" rel="import">
+<link href="../layout/layout.html" rel="import">
+<dom-module id="paper-shadow">
+  <style>
+    paper-shadow {
+      display: block;
+      position: relative;
+      pointer-events: none;
+    }
+
+    paper-shadow #shadow-bottom,
+    paper-shadow #shadow-top {
+      border-radius: inherit;
+    }
+
+    paper-shadow #shadow-bottom[animated],
+    paper-shadow #shadow-top[animated] {
+      transition: box-shadow 0.28s cubic-bezier(0.4, 0, 0.2, 1);
+    }
+
+    paper-shadow #shadow-top.paper-shadow-z-1 {
+      box-shadow: none;
+    }
+
+    paper-shadow #shadow-bottom.paper-shadow-z-1 {
+      box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.37);
+    }
```

Comment:
I can't remember why I made the top and bottom shadow elements separate, but you can do `box-shadow: [topShadow], [bottomShadow]` on the host element. I think I made the shadow element separate because I didn't want to add a class to the host, which doesn't play well with the the user binding to `class`, but it seems to simplify things to style the host directly.

Maybe an option is to (gasp) use attributes to apply the style (for example paper-shadow[z=1]) or use inline styles.